lib/git-lfs-s3/application.rb

```diff
@@ -101,7 +101,7 @@ def public_read_grant
         status 404
       end
       if settings.public_server and settings.ceph_s3
-        if object.acl.grants.include?(public_read_grant)
+        if not object.acl.grants.include?(public_read_grant)
```

Comment:
This makes sense to me but I'm not clear as it why this was working at all before?